src/main/java/Task.java

```diff
@@ -0,0 +1,55 @@
+import java.io.Serializable;
```

Comment:
Is this import used for the cross and tick symbols? Otherwise, I can't really see where it is used. Perhaps you could remove it? Apologies if I missed out on its usage!